README.md

```diff
@@ -4,37 +4,40 @@
 <br>
 
-ReviOS is a customized version of Windows, designed to be lightweight, stable, and focused on improving performance, privacy and compatibility. It aspires to re-create what Windows as an operating system should have been - easy and simple. Now compatible with [AME Wizard](https://ameliorated.io/).
+ReviOS is a customized version of Windows, designed to be lightweight, stable, and focused on improving performance, privacy and compatibility. It aspires to re-create what Windows as an operating system should have been - easy and simple. Now compatible with the [AME Wizard](https://ameliorated.io).
 
 ## Features
 
-- Easy to use, just drag and drop
+- Automaticlly built by GitHub actions for reduced release times
+- Easy to use, just drag and drop the playbook into the [AME Wizard](https://ameliorated.io)
 - Removes bloatware and unnecessary system components
-- Tweaks your system to optimize performance and improve privacy
-- Integrates [Revision Tool](https://github.com/meetrevision/revision-tool) to customize your system further
+- Optimizes your system for performance and improved privacy
+- Integrates the [Revision Tool](https://github.com/meetrevision/revision-tool) to customize your system further
 
 ## System Requirements
 
-- Windows 10 21H2 `19044` (LTSC)
-- Windows 10 22H2 `19045`
-- Windows 11 22H2 `22621`
+- Windows 10 21H2 [`19044`](https://en.wikipedia.org/wiki/Windows_10_version_21H2) (LTSC)
+- Windows 10 22H2 [`19045`](https://en.wikipedia.org/wiki/Windows_10_version_22H2)
+- Windows 11 22H2 [`22621`](https://en.wikipedia.org/wiki/Windows_11_version_22H2)
 
-Any other build isn't supported by Revision.
+Any other build **isn't** supported by Revision.
 
-## Get Started
+## Getting Started
 
-To use ReviOS Playbook on your machine, follow these steps:
+To use ReviOS Playbook on your computer, follow these steps:
 
-1. Download [AME Wizard](https://ameliorated.io/) and our playbook from [releases](https://github.com/meetrevision/playbook/releases)
-2. Open AME Wizard and drag and drop the downloaded Playbook file into the drag box
+1. Download the [AME Wizard](https://ameliorated.io) and our playbook from [releases](https://github.com/meetrevision/playbook/releases)
+2. Run the AME Wizard
+3. Drag and drop the downloaded playbook into the drag box
+4. Follow the on-screen instructions
 
 ## Documentation
 
-For more information, check out our documentation:
+See our documentation for more information:
 
 - [Playbook](https://www.revi.cc/docs/playbook)
-- [Frequently Asked Question](https://www.revi.cc/docs/faq)
+- [Frequently Asked Questions](https://www.revi.cc/docs/faq)
 
 ## License
 
-ReviOS Playbook is licensed under [Attribution-NonCommercial-NoDerivatives 4.0 International](https://creativecommons.org/licenses/by-nc-nd/4.0/)
\ No newline at end of file
+The ReviOS Playbook is licensed under [Attribution-NonCommercial-NoDerivatives 4.0 International.](https://creativecommons.org/licenses/by-nc-nd/4.0/)
```

Comment:
while we're at it: the period does not belong inside the link, it goes after the link.

Reply:
fixed